README.md

```diff
@@ -1,2 +1,70 @@
-# gks-clinvar
-A submission tool for submitting to the ClinVar Submission API using GA4GH GKS data 
+# GKS-ClinVar
+
+<!-- [![image](https://img.shields.io/pypi/v/gks-clinvar.svg)](https://pypi.python.org/pypi/gks-clinvar)
+[![image](https://img.shields.io/pypi/l/gks-clinvar.svg)](https://pypi.python.org/pypi/gks-clinvar)
+[![image](https://img.shields.io/pypi/pyversions/gks-clinvar.svg)](https://pypi.python.org/pypi/gks-clinvar)
+[![Actions status](https://github.com/ga4gh/gks-clinvar/actions/workflows/checks.yaml/badge.svg)](https://github.com/ga4gh/gks-clinvar/actions/checks.yaml) -->
+
+A submission tool for submitting to the ClinVar Submission API using [GA4GH Genomic
+Knowledge Standards](https://www.ga4gh.org/work_stream/genomic-knowledge-standards/)
+(GKS) data.
+
+Development of a shared, open-source software package for managing these submissions
+that can be easily configured for use by multiple submitters would be highly impactful
+as development and adoption of the GKS framework grows.
+
+![image](./misc/images/GKS-ClinVar%20submission%20overview.png)
+
+We plan to contribute to [ClinVar This](https://github.com/varfish-org/clinvar-this) to
+support GKS data stored as [JSON Lines](https://jsonlines.org/) files.
```

Comment:
General overview. Should I add anything else?

Reply:
I think this looks great.